src/main/java/org/wildfly/security/pem/Pem.java

```diff
@@ -48,8 +50,9 @@
  */
 public final class Pem {
     private static final Pattern VALID_LABEL = Pattern.compile("[^ -~&&[^-]]");
-    private static final String PUBLIC_KEY_FORMAT = "PUBLIC KEY";
+    private static final String PUBLIC_KEY_FORMAT = "PUBLICKEY";
```

Comment:
PKCS#8 uses a space here, and PKCS#1 further gives e.g. "RSA PUBLIC KEY".  We need to support all the standard formats explicitly, and we also don't want to accept invalid input (e.g. "PUBLIC     KEY").